### ISSUE TITLE
npm prerequisite 'npm not listed in documentation

### ISSUE THREAD

**Issue by childers** [GMOD/Apollo#1135]
I was just checking out 2.0.3 and got this error below.  Should npm be added to the list of required software?
```shell
$ ./apollo clean
Defining custom 'build' task when using the standard Gradle lifecycle plugins has been deprecated and is scheduled to be removed in Gradle 3.0
final JBrowse settings [git:[url:https://github.com/gmod/jbrowse, tag:1.12.2-apollo, alwaysPull:false, alwaysRecheck:false], plugins:[WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
final plugins [WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]
Copying apollo plugin
handling jbrowse release [git:[url:https://github.com/gmod/jbrowse, tag:1.12.2-apollo, alwaysPull:false, alwaysRecheck:false], plugins:[WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
handling jbrowse [git:[url:https://github.com/gmod/jbrowse, tag:1.12.2-apollo, alwaysPull:false, alwaysRecheck:false], plugins:[WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]

FAILURE: Build failed with an exception.

* Where:
Build file './build.gradle' line: 371

* What went wrong:
Execution failed for task ':npmBin'.
> A problem occurred starting process 'command 'npm''

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 7.465 secs
```

**Comment by nathandunn** [comment]
Yes!

**Comment by nathandunn** [comment]
Sould be fixed by #1136.
